{"name": "test_bas_result_check", "status": "failed", "statusDetails": {"message": "AssertionError: assert True == False\n  +True\n  -False", "trace": "test_bad_result.py:3: in test_bas_result_check\n    assert True == False\nE   assert True == False\nE     +True\nE     -False"}, "attachments": [{"name": "log", "source": "b5db2d1f-b213-4e6f-8c0d-d51a670f7762-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "39843995-65ef-4c93-81ed-cc7ca599d93f-attachment.txt", "type": "text/plain"}], "start": 1596530589763, "stop": 1596530589763, "uuid": "492274bf-393d-4b4b-a69d-1ed92dfdb6b3", "historyId": "b90642ceb2f3b4a90b5166f617bed14a", "testCaseId": "901d9abe59f0ff217281fd5a08fe93da", "fullName": "test_bad_result#test_bas_result_check", "labels": [{"name": "suite", "value": "test_bad_result"}, {"name": "host", "value": "S340"}, {"name": "thread", "value": "31433-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_bad_result"}]}